Docs - Improve SEO (#99)
iotaledger/IOTA-2.0-Research-Specifications: docs/5.3Mana.md

---

> description: This section introduces  *access Mana* and *consensus Mana* and determines which modules use each of these two types of Mana. 

Reviewer: ```suggestion
description: This section introduces  *access Mana* and *consensus Mana* and explains which modules use which Mana. 
```
